src/ui/server/api.ts

```diff
@@ -73,6 +74,29 @@ const keystone = {
       return response.data.page;
     }
 
+    return {};
+  },
+  getCompaniesHouseInformation: async (companiesHouseNumber: string) => {
+    const queryParams = {
+      companiesHouseNumber,
+    };
+
+    const query = companiesHouseQuery;
+    const response = (await apollo('GET', query, queryParams)) as ApolloResponse;
+
+    if (response.errors) {
+      console.error('GraphQL error querying keystone page ', response.errors);
+    }
+
+    if (response?.networkError?.result?.errors) {
+      console.error('GraphQL network error querying keystone page ', response.networkError.result.errors);
```

Comment:
`keystone page`

FYI there will be merge conflicts here soon and we should follow try/catch approach (not merged yet): https://github.com/UK-Export-Finance/exip/blob/main-application/e2e-tests/cypress/e2e/journeys/insurance/eligibility/exporter-location/exporter-location.spec.js

Reply:
Need to replace `GraphQL network error querying keystone page` with 

`GraphQL network error querying companies house information`

Reply:
fixed